apps/api/src/graphql/schema/programs.ts

```diff
@@ -76,6 +76,22 @@ type UgradRequirements @cacheControl(maxAge: 86400) {
     requirements: [ProgramRequirement!]!,
 }
 
+type Curriculum {
```

Comment:
i think you have to `cacheControl` these inner types too but i might be wrong, look into it yourself and change if needed